data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -137,5 +143,15 @@ while true; do
             exit 1
         fi
     done
+
+    # See if all 3 masters are up yet - if so we can bring down Ironic. If the masters are brought up, and
+    # machine-api is started in the cluster, there can end up being 2 DHCP servers running on the network.
+    # We must ensure we stop the bootstrap's Ironic before that can happen.
+    ACTIVE_NODES=$(curl -H "X-OpenStack-Ironic-API-Version: 1.9" 'http://localhost:6385/v1/nodes?provision_state=active' | jq '.nodes | length')
+    if [[ "$ACTIVE_NODES" == "3" ]]; then
```

Comment:
If we go with this approach we'll need to template the number of nodes as I know some folks have tested with a single master